features/car/restrictions.feature

```diff
@@ -430,3 +430,27 @@ Feature: Car - Turn restrictions
             | a    | b  | ax,xy,yb,yb |
             | b    | a  | yb,xy,ax,ax |
 
+    @specific
+    Scenario: Car - Ignore unrecognized restriction
+        Given the node map
+            |   | n |   |
+            | w | j | e |
+            |   | s |   |
+
+        And the ways
+            | nodes | oneway |
+            | sj    | yes    |
+            | nj    | -1     |
+            | wj    | -1     |
+            | ej    | -1     |
+
+        And the relations
+            | type        | way:from | way:to | node:via | restriction  |
+            | restriction | sj       | wj     | j        | yield        |
```

Comment:
can we also add tests that start with only_ stuff like `only_if_you_are_lucky` or `no_not_right_now` to make sure we only use recognised tags?